CONTRIBUTING.md

```diff
@@ -1,74 +1,53 @@
-
-* Check that there is an existing issue in GitHub issues
-* Check if there are other pull requests that might overlap or conflict with your intended contribution
-
-# How to Contribute
-
-1. Fork the repository
-2. Make some changes on a branch on your fork
-3. Create a pull request from your branch
-
-In your pull request, outline:
-
-* What the changes intend
-* How they change the existing code
-* If (and what) they breaks
-* Start the pull request with the GitHub issue ID, e.g. #123
-
-Lastly, please follow the [pull request template](.github/pull_request_template.md) when submitting a pull request!
-
-Each commit message that is not part of a pull request:
-
-* Should contain the issue ID like `#123`
-* Can contain the tag `[trivial]` for trivial changes that don't relate to an issue
-
-
-
-## Coding Styleguides
-
-We enforce a coding styleguide using `eslint`. As part of your build, run `npm run lint` to check if your code is conforming to the style guide. We do the same for every PR in our CI, so PRs will get rejected if they don't follow the style guide.
-
-You can fix some of the issues automatically by running `npx eslint . --fix`.
-
-## Commit Message Format
-
-This project uses a structured commit changelog format that should be used for every commit. Use `npm run commit` instead of your usual `git commit` to generate commit messages using a wizard.
-
-```bash
-# either add all changed files
-$ git add -A
-# or selectively add files
-$ git add package.json
-# then commit using the wizard
-$ npm run commit
-```
-
-# How Contributions get Reviewed
-
-One of the maintainers will look at the pull request within one week. Feedback on the pull request will be given in writing, in GitHub.
-
-# Release Management
-
-The project's committers will release to the [Adobe organization on npmjs.org](https://www.npmjs.com/org/adobe).
-Please contact the [Adobe Open Source Advisory Board](https://git.corp.adobe.com/OpenSourceAdvisoryBoard/discuss/issues) to get access to the npmjs organization.
-
-The release process is fully automated using `semantic-release`, increasing the version numbers, etc. based on the contents of the commit messages found.
+# Contributing to BACOM
+
+## Permissions
+Internal employees can reach out to the team on [slack](https://adobe-mwp.slack.com/archives/C03S7CZU8NN) to get github permissions to contribute to bacom.
+External community members who want to contribute can fork the repo.
+
+## Local Setup
+Once you've pulled down the repo, follow the local setup instructions included in the [README](/README.md).
+
+## Authoring Experience
+We strongly recommend reaching out to the milo community for feedback on your prospective authoring experience before beginning development. This will help avoid AX issues being flagged in code review which could necessitate a lengthy rewrite. You can post your AX ideas in [slack](https://adobe-mwp.slack.com/archives/C03B0BUA75E) to receive feedback. Screenshots are always appreciated.
```

Comment:
`...prospective authoring experience(AX)...` add parenthesis and acronym to make the acronym more clear since the audience for this seems general and the acronym is used more than once.